NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml

```diff
@@ -0,0 +1,672 @@
+                                Text="Blur R"
+                            />
+                            <nui:Slider x:Name="ShadowBlurRadius"
+                                        MinValue="0"
+                                        MaxValue="50"
+                                        CurrentValue="10"
+                                        ThumbSize="8,8"
+                                        WidthSpecification="-1"
+                            />
+                        </b:View>
+
+                        <b:View
+                            x:Name="ShadowToggleView2"
+                            WidthSpecification="-1"
+                            HeightSpecification="-2"
+                        >
+                            <View.Layout>
+                                <LinearLayout LinearOrientation="Horizontal" LinearAlignment="CenterVertical" CellPadding="5,5"/>
+                            </View.Layout>
+                            <b:TextLabel
+                                x:Name="ShadowToggleButtonT"
+                                PointSize="15"
+                                HorizontalAlignment="Begin"
+                                VerticalAlignment="Center"
+                                Text="Toggle HasShadow:"
+                            />
+                            <nui:CheckBox x:Name="ShadowToggleButton"
+                                          IsSelected="True"
+                            />
+                        </b:View>
+                     </b:View>
+                </b:View>
+
+                <!-- Shadow Color -->
+                <b:View
+                    x:Name="ShadowColorView"
+                    WidthSpecification="-1"
+                    HeightSpecification="-2"
+                >
+                    <View.Layout>
+                        <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                    </View.Layout>
+                    <b:TextLabel
+                        x:Name="ShadowColorViewHeader"
+                        WidthSpecification="-1"
+                        HorizontalAlignment="Begin"
+                        VerticalAlignment="Center"
+                        PointSize="25"
+                        TextColor="#fe7c57"
+                        Text="ShadowColor"
+                    />
+                    <b:View
+                        x:Name="ShadowColorViewContent"
+                        WidthSpecification="-1"
+                        HeightSpecification="-2"
+                    >
+                        <View.Layout>
+                            <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                        </View.Layout>
+                        <b:View
+                            x:Name="ShadowColorView1"
+                            WidthSpecification="-1"
+                            HeightSpecification="-2"
```

Comment:
use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

.... all negative numeric need to be changed this. thank you.